.changeset/mean-tomatoes-shout.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-permission-backend': patch
+'@backstage/plugin-permission-react': patch
+'@backstage/plugin-search-backend': patch
+---
+
+Use updated types from `@backstage/plugin-permission-common`
```

Comment:
Is this changeset necessary considering the automated versioning process will already bump these? Related, does `@backstage/plugin-catalog-backend` need to be included since the api-report changed?

Reply:
Yup makes sense to me!